dask/array/tests/test_ufunc.py

```diff
@@ -237,6 +237,7 @@ def test_ufunc_outer():
 def test_complex(ufunc):
 
     dafunc = getattr(da, ufunc)
+    # Note that these functions are not NumPy ufuncs ...
```

Comment:
@pentschev not sure if this matters or not, but these 4 functions aren't ufuncs. I just did the minimal fix below, but maybe the test needs moving?

Reply:
There's more functions in `ufunc.py` that are not ufuncs but are handled with `__array_function__`. Should those be moved? They're now explicitly labeled as ufuncs (all uses of `wrap_elemwise`), which is very confusing....

Reply:
I believe they were added in `test_ufunc.py` because they're handled in `ufunc.py`. Maybe they could be moved elsewhere, from a naming point-of-view, I would say `wrap.py`, but that's also something different. So I would suggest either `wrap.py` or creating a new file. I imagine the reason they were added to `ufunc.py` was because whoever added those wasn't sure where to put them.